src/briefcase/commands/dev.py

```diff
@@ -50,6 +50,11 @@ def add_options(self, parser):
             action="store_true",
             help='Update dependencies for app'
         )
+        parser.add_argument(
+            '--no-run',
+            action="store_true",
+            help='Do not run the app, just install dependencies.'
+        )
```

Comment:
Although `no-run` is the option we want, having the negative in the stored argument name is cumbersome (as it leads to logic like "if not no_run"). If you add `dest='run_app', action='store_false'` here, you can check for `args.run_app`, which will be true by default, and false if the `--no-run` flag is provided.